### PR TITLE
Issue #111: Adoptium actions adds source

### PR DESCRIPTION
**Pull request by anthonydahanne**
## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Review by dmikusa — APPROVED** [review]

File: actions/adoptium-dependency/main.go [hunk 1]
```diff
@@ -70,16 +70,18 @@ func main() {
 	}
 
 	versions := make(actions.Versions)
+	additionalOutputs := make(actions.Outputs)
 	for _, r := range raw {
 		versions[strings.ReplaceAll(r.VersionData.Semver, "+", "-")] = r.Binaries[0].Package.Link
+		additionalOutputs["source"] = r.Source.Link
```
Comment (pivotal-david-osullivan) [inline actions/adoptium-dependency/main.go:76]: Unless I'm missing something - the source URI here is not linked to the release/asset version that we are looping. This results in a source URI that doesnt match the latest version.

In other actions, I used the looping version as the key to the source map. Then when we calculate Latest, we use that value to pull the corresponding source from the map. See https://github.com/paketo-buildpacks/pipeline-builder/blob/fea2141ee5a3665301b3ec35636ff35984de37fd/actions/tomcat-dependency/main.go#L52-L53

Comment (anthonydahanne) [reply actions/adoptium-dependency/main.go:76]: oups, yes indeed!

**Comment by anthonydahanne** [comment]
@pivotal-david-osullivan fixed!

**Comment by anthonydahanne** [comment]
@pivotal-david-osullivan I have not waited for your second feedback; but I believe I stuck to your recommendations!

**Pull request merged**